v2/util/time.go

```diff
@@ -54,6 +54,7 @@ var (
 	OnionOnlyEVDate             = time.Date(2015, time.May, 1, 0, 0, 0, 0, time.UTC)
 	CABV201Date                 = time.Date(2017, time.July, 28, 0, 0, 0, 0, time.UTC)
 	AppleCTPolicyDate           = time.Date(2018, time.October, 15, 0, 0, 0, 0, time.UTC)
+	MozillaPolicy21Date         = time.Date(2013, time.February, 1, 0, 0, 0, 0, time.UTC)
```

Comment:
I don't see this new exported constant referenced anywhere in this branch. Should it be removed?

Reply:
@cpu It was referenced in an older version. Since it is correct, I don't see a reason to remove it, as it might be needed later by someone else. But of course, I could also remove it. As you like.

Reply:
@RufusJWB Unless/until it's needed, it's better to remove. YAGNI and such :)

Reply:
I removed it.